server/services/arduino/lib/device.configure.js

```diff
@@ -0,0 +1,48 @@
+const { DEVICE_FUNCTION } = require('../../../utils/constants');
+
+const logger = require('../../../utils/logger');
+
+/**
+ * @description Change value of a device
+ * @param {Object} device - The device to configure.
+ * @returns {Promise} - .
+ * @example
+ * configure(device);
+ */
+async function configure(device) {
+  const arduino = await this.gladys.device.getBySelector(
+    device.params.find((param) => param.name === 'ARDUINO_LINKED').value,
+  );
+  const path = arduino.params.find((param) => param.name === 'ARDUINO_PATH').value;
+
+  let functionName = device.params.find((param) => param.name === 'FUNCTION').value;
+
+  if (functionName === DEVICE_FUNCTION.DHT_HUMIDITY || functionName === DEVICE_FUNCTION.DHT_TEMPERATURE) {
+    functionName = 'recv_dht';
+  }
+
+  const message = {
+    function_name: functionName,
+    parameters: {
+      data_pin: device.params.find((param) => param.name === 'DATA_PIN').value,
+    },
+  };
+
+  switch (functionName) {
+    case 'recv_dht':
```

Comment:
Why this value is not in the DEVICE_FUNCTION constants?

Reply:
Because for DHT, there are two types of data : temperature and humidity. So in the DEVICE_FUNCTION, we have this distinction in order to know which data will be returned, and `recv_dht` is used as the function in the .ino file to configure reception (data pin, reception enabled ?).

I made some changes in this file, maybe it will be more clear ^^